js/gc-smart-data.js

```diff
@@ -406,7 +406,10 @@ window.GC = window.GC || {};
         if ( d.valid() ) {
             this.DOB = d;
             this.birthdate = d.toString();
-            this.gestationAge = this.weeker = Math.round(40 - this.DOB.diffWeeks(this.EDD));
+
+            if (this.gestationAge == null) {
```

Comment:
Could you explain why do you want to keep the gestationAge/weeker if DOB or EDD is changed?